pootle/apps/accounts/management/commands/find_duplicate_emails.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from accounts.utils import get_duplicate_emails
+
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        duplicates = get_duplicate_emails()
+        if not duplicates:
+            print "There are no accounts with duplicate emails"
+            return
+
+        for email in duplicates:
+            users = User.objects.filter(email=email)
+            if not email:
```

Comment:
How is this scenario even possible? I mean, if you have a list of duplicate emails `email` here should have some value, isn't it?